src/Components/App/App.js

```diff
@@ -8,32 +8,35 @@ import './App.css';
 
 class App extends Component {
   constructor() {
-    super();
+    super()
     this.state = {
       movies: [],
       error: '',
       singleMovie: null,
       bannerImage: ''
-    };
+    }
   }
 
   componentDidMount = () => {
     fetchAllMoviesData()
       .then((movieData) => {
-        const randomMovie = movieData.movies[Math.floor(Math.random() * movieData.movies.length)]
-
-        this.setState({ movies: movieData.movies, bannerImage: randomMovie.backdrop_path })
+        this.setState({
+          movies: movieData.movies,
+          bannerImage: this.getRandomMovieImage(movieData.movies)
+        })
       })
-      .catch((error) => this.setState({ error: error.message }));
-  };
+      .catch((error) => this.setState({ error: error.message }))
+  }
 
   selectMovie = (id) => {
     fetchSingleMovieData(id)
-      .then((singleMovieData) =>
-        this.setState({ singleMovie: singleMovieData.movie })
-      )
-      .catch((error) => this.setState({ error: error.message }));
-  };
+      .then((singleMovieData) => this.setState({ singleMovie: singleMovieData.movie }))
+      .catch((error) => this.setState({ error: error.message }))
+  }
+
+  getRandomMovieImage = (movies) => {
```

Comment:
This is a great feature, I like how to movie changes every time you load the page

Reply:
I agree - cool feature!